docs/usage/self-hosted-configuration.md

```diff
@@ -555,13 +555,89 @@ It also may mean that ignored directories like `node_modules` can be preserved a
 
 ## platform
 
+## postUpgradeTasks
```

Comment:
no, this needs to be moved back to other config docs, as it's a repo config option.

only options with global flag should be defined here.

Reply:
Sorry about giving the wrong instructions! 🙈

Reply:
@HonkingGoose no problem, I just need to revert commit :)